js/languages/en-US.json

```diff
@@ -21,7 +21,8 @@
   },
   "connectedPeersPage": {
     "heading": "Connected Peers",
-    "infoMessage": "Please note: If any peer does not have their profile set, clicking on their guid below will result in a User Not Found page.",
+    "totalPeers": "%{totalPeers} Found",
+    "infoMessage": "If you do not see any peers below, you are not connected to the OpenBazaar network. If a peer does not have information available on the network, clicking on their guid below will result in a User Not Found page.",
```

Comment:
> If you do not see any peers below, you are not connected to the OpenBazaar network.

This will never be applicable to the user because if they are not connected to a server, the Connection Management modal will be up and obscuring everything else.

With that said, should we remove that part and save the user the cognitive load of a case that will never be true for them?